cds/modules/deposit/static/js/cds_deposit/avc/components/cdsDeposits.js

```diff
@@ -436,6 +424,58 @@ function cdsDepositsCtrl(
       );
     }
   );
+
+  this.broadcastEvent = function (eventName, args) {
+    $scope.$broadcast(eventName, args);
+  };
+
+  this.helpers = (function () {
+
+      function containsLink(links, link) {
```

Comment:
hmm I'm just wondering is it any specific reason to introduce closures?

Reply:
Well, the idea is to have stateless ("static") methods here, helpers, not related to any controller or scope, and to encapsulate. Keeping a closure, should help to have better code readability, should avoid to have devs. to use `that` inside, and should simplify refactoring, by moving out that helpers in a different module/factory/service/whatever.
It is a way, IMHO, to say: "hey, this is not stuff of the controller".

Then, for that specific code, we can have it in the scope of the controller. I can change it and remove the closure if you prefer! :)

Reply:
👍 thanks for the explanation. IMHO we can remove it as we don't use closures in the module. 
 
WDYT about moving it to ``cdsAPI`` factory which is already contains related helpers i.e. ``chainedActions`` etc. :)

Reply:
Fixed as you suggested... I find confusing now having `action` and `makeAction` in cdsAPI, but we will change everything in the future, so...